front-end/studio/src/app/pages/apis/apis.page.ts

```diff
@@ -78,7 +78,7 @@ export class ApisPageComponent extends AbstractPageComponent implements OnDestro
     public allApis: Api[] = [];
     public filteredApis: Api[];
     public selectedApis: Api[];
-    public filters: Filters = new Filters();
+    private filters: Filters = new Filters();
```

Comment:
Any particular reason to make this particular field private?  I'm sure there are lots of fields around that could be private but are not.

Reply:
I can't remember exactly, but I think the ahead of time compilation failed as I haven't imported `Filters` in the `public_api.ts`, being that `Filters` is only used here made more sense to make it `private`.